src/net/mod.rs

```diff
@@ -28,9 +28,103 @@
 //! # }) }
 //! ```
 
-pub use tcp::{Incoming, TcpListener, TcpStream};
+pub use tcp::Incoming;
 pub use udp::UdpSocket;
 
 pub(crate) mod driver;
 mod tcp;
 mod udp;
+
+use cfg_if::cfg_if;
+
+cfg_if! {
```

Comment:
No need to add this section for `TcpListener` because ours differs from the one in `std`

Reply:
The same applies to `TcpStream`